src/black/strings.py

```diff
@@ -236,3 +246,30 @@ def normalize_string_quotes(s: str) -> str:
         return s  # Prefer double quotes
 
     return f"{prefix}{new_quote}{new_body}{new_quote}"
+
+
+def normalize_unicode_escape_sequences(leaf: Leaf) -> None:
+    """Replace hex codes in Unicode escape sequences with lowercase representation."""
```

Comment:
This will have to be thought out still, as [this comment](https://github.com/psf/black/issues/2067#issuecomment-1024993814) points out. My two cents: I prefer upper case, and since Black formats hex numbers to upper already I think it would be consistent. The Python repr argument is solid too, but we should think about changing hex literals as well then.

Reply:
I'd rather not change hex numbers, we already changed our mind there a few times.

Reply:
So if we're not changing numbers (which I agree with), do y'all share the concern for consistency?

Reply:
My comments read a bit ambiguously. So to be clear, I'm proposing that we switch the formatting to be upper case to be consistent with hex numbers. Y'all in?